BaragonService/src/main/java/com/hubspot/baragon/service/resources/LoadBalancerResource.java

```diff
@@ -68,13 +70,14 @@ public LoadBalancerResource(BaragonLoadBalancerDatastore loadBalancerDatastore,
   @POST
   @Path("/{clusterName}/sources")
   public BaragonGroup addSource(@PathParam("clusterName") String clusterName, @QueryParam("source") String source) {
-    return loadBalancerDatastore.addSourceToGroup(clusterName, source);
+    TrafficSource trafficSource = new TrafficSource(source, TrafficSourceType.CLASSIC);
+    return loadBalancerDatastore.addSourceToGroup(clusterName, trafficSource);
   }
 
```

Comment:
Can we add an additional `POST` method here that takes a body of a `TrafficSource` object? Would be the same as the `/{clusterName}/sources` one above, but different args.